ivadomed/inference.py

```diff
@@ -275,15 +308,14 @@ def segment_volume(folder_model, fname_images, gpu_id=0, options=None):
         list: List of target suffix associated with each prediction in `pred_list`
 
     """
-    # Define device
-    cuda_available = torch.cuda.is_available()
-    device = torch.device("cpu") if not cuda_available else torch.device("cuda:" + str(gpu_id))
 
-    # Check if model folder exists and get filenames
-    fname_model, fname_model_metadata = imed_models.get_model_filenames(folder_model)
+    # Check if model folder exists and get filenames to be stored as string
+    path_model_file: str
```

Comment:
Is `fname_model` replaced by `path_model_file`? I'm curious why the name changed?

Reply:
just so @dyt811 is aware of [our naming conventions](https://github.com/spinalcordtoolbox/spinalcordtoolbox/wiki/Programming%3A-Naming-conventions-(variables,-files)#variables)

Reply:
Cool. TIL. I reverted it back and will be watching out for that going forward. Definitely shorter than my typical path_to_file_reason_x_y_z style. I didn't see that in IvadoMed wiki so I must have missed a meeting on the issue. Should I copy SCT's guideline over the IvadoMed on these conventions?

Reply:
> Should I copy SCT's guideline over the IvadoMed on these conventions?

i'm always reluctant to duplicate information (which implies: more than one place to maintain). We might want to centralize our conventions and redirect instead. @joshuacwnewton had thought about it a lot so i'm tagging him here

Reply:
> (which implies: more than one place to maintain)

FYI: My opinions have shifted a little, ever since trying to coordinate the contributing guidelines.

I found it takes just as much effort (perhaps even more) to have to maintain a centralized guide. Someone has to coordinate projects to agree on shared guidelines, make sure the recommendations are general enough to suit every project, create links to point at the centralized guide, and ensure developers know to go look at this external guide (that alone is a big ask). 

So, for smaller things like variable naming conventions, I've started to feel that it's OK (even preferable) to duplicate the info for each project-specific Wiki, just to keep the info close by for each individual project.

Reply:
thank you for your insights @joshuacwnewton ! it makes perfect sense.

Reply:
so it means we should probably copy some of the parts of the SCT's wiki into ivadomed, @dyt811 if you want to help with that

Reply:
Will do. Absolutely makes sense. Will copy it over to clarify this point and review the SCT guideline in depth as well.